### DISCUSSION TITLE
Where to buy Adapt-FFB-Joy adapters?

### DISCUSSION THREAD

**Discussion by Hedda** [tloimu/adapt-ffb-joy#41] [General]
Does anyone know where we can buy preassembled Adapt-FFB-Joy adapters?  Preferably with shipping from inside the Europe Union.

PS: Preassembled SidewinderToUSBV2 looks like a nice but more complicated option https://github.com/Kreeblah/SidewinderToUSBV2

**Comment by aleraco** [comment]
Hi! Please write to aleraco@gmail.com
Microsoft Sidewinder Force Feedback pro USB adapter with force feedback
https://youtu.be/q7qJgmT5FQM
i have boards available and i send from Italy
